diktat-gradle-plugin/src/main/kotlin/org/cqfn/diktat/plugin/gradle/Utils.kt

```diff
@@ -0,0 +1,54 @@
+/**
+ * Utilities for diktat gradle plugin
+ */
+
+package org.cqfn.diktat.plugin.gradle
+
+import groovy.lang.Closure
+
+/**
+ * A wrapper for gradle version, which supports three-digit versions like `6.6.1` or `6.0`
+ * with possible suffix: `6.6.0-M1`, `6.6-RC`
+ *
+ * @property major major version
+ * @property minor minor version
+ * @property patch patch version
+ * @property suffix version suffix
+ */
+internal data class GradleVersion(
+        val major: Int,
+        val minor: Int,
+        val patch: Int,
+        val suffix: String?) {
+    companion object {
+        /**
+         * @param version string representation of version, e.g. from [org.gradle.api.invocation.Gradle.getGradleVersion]
+         * @return a [GradleVersion]
+         */
+        fun fromString(version: String): GradleVersion {
+            val (versionDigits, suffix) = version.split('-', limit = 2).run {
+                first() to getOrNull(1)
```

Comment:
please specify explicitly  that you are using getOrNull(1) on the split-version.
Now it is less readable